modules/apps/depot/depot-web/src/main/resources/META-INF/resources/screen/navigation/entries/documents_and_media.jsp

```diff
@@ -30,4 +30,25 @@ UnicodeProperties typeSettingsProperties = group.getTypeSettingsProperties();
 	label='<%= LanguageUtil.get(request, "documents-and-media") %>'
 >
 	<aui:input helpMessage='<%= LanguageUtil.format(request, "can-user-with-view-permission-browse-the-asset-library-document-library-files-and-folders", new Object[] {HtmlUtil.escape(group.getDescriptiveName(themeDisplay.getLocale())), themeDisplay.getPortalURL() + "/documents" + group.getFriendlyURL()}, false) %>' inlineLabel="right" label="enable-directory-indexing" labelCssClass="simple-toggle-switch" name="TypeSettingsProperties--directoryIndexingEnabled--" type="toggle-switch" value='<%= PropertiesParamUtil.getBoolean(typeSettingsProperties, request, "directoryIndexingEnabled") %>' />
+
+	<liferay-util:html-top>
+		<link href="<%= PortalUtil.getStaticResourceURL(request, PortalUtil.getPathModule() + "/document-library-web/document_library/css/main.css") %>" rel="stylesheet" />
+	</liferay-util:html-top>
+
+	<liferay-frontend:fieldset
+		cssClass="mt-5"
+		collapsible="<%= false %>"
+		label='<%= LanguageUtil.get(request, "maximum-file-size-and-mimetypes") %>'
+	>
+		<div>
```

Comment:
Maybe we need some placeholder rendered at JSP before react component render.

Reply:
I'll take a look at this. Actually, I'm not a big fan of having placeholders in every piece of JSP that renders a react component just because of the "flash", specially if this is not perceived by the user (because it is not visually at the beginning, i.e you have to scroll), and because we are duplicating code and later we have to maintain it.

Reply:
We don't need to duplicate the code, maybe a loader is enough.